iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -182,6 +182,7 @@ public class HiveIcebergStorageHandler implements HiveStoragePredicateHandler, H
   public static final String COPY_ON_WRITE = "copy-on-write";
   public static final String MERGE_ON_READ = "merge-on-read";
   public static final String STATS = "/stats/";
+  static final String WRITE_KEY = "HiveIcebergStorageHandler_write";
```

Comment:
Here i use this term to indicate the table is an output table and this term was previously removed by HIVE-26102. If you have any better suggestion, feel free to comment.

Reply:
how about simply `iceberg.mr.output.table`?

Reply:
Changed this line to `static final String WRITE_KEY = "iceberg.mr.output.table";`